From stern@rowland.harvard.edu Mon Nov 28 12:26:20 2005
Date: Mon, 28 Nov 2005 15:22:55 -0500 (EST)
From: Alan Stern <stern@rowland.harvard.edu>
To: Greg KH <greg@kroah.com>
cc: David Brownell <david-b@pacbell.net>, Chris Humbert <mahadri-usb@drigon.com>
Subject: USB: documentation update
Message-ID: <Pine.LNX.4.44L0.0511281521290.4910-100000@iolanthe.rowland.org>

This patch (as611) fixes a minor mistake and misspelling in the USB 
documentation.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/usb/error-codes.txt |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/Documentation/usb/error-codes.txt
+++ gregkh-2.6/Documentation/usb/error-codes.txt
@@ -46,8 +46,9 @@ USB-specific:
 
 -EMSGSIZE	(a) endpoint maxpacket size is zero; it is not usable
 		    in the current interface altsetting.
-		(b) ISO packet is biger than endpoint maxpacket
-		(c) requested data transfer size is invalid (negative)
+		(b) ISO packet is larger than the endpoint maxpacket.
+		(c) requested data transfer length is invalid: negative
+		    or too large for the host controller.
 
 -ENOSPC		This request would overcommit the usb bandwidth reserved
 		for periodic transfers (interrupt, isochronous).
